app/generic/webhooks.py

```diff
@@ -0,0 +1,40 @@
+import logging
+from typing import List, Union
+from aries_cloudcontroller import AcaPyClient
+
+from fastapi import APIRouter, Depends
+
+from app.dependencies import agent_selector
+from app.facades.webhooks import (
+    get_hooks_per_topic_admin,
+    get_hooks_per_topic_per_wallet,
+    topics,
+)
+from app.generic.models import ProofsHook, ConnectionsHook, CredentialsHooks, TopicItem
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter(prefix="/webhooks", tags=["webhooks"])
+
+
+@router.get("/{topic}")
+async def get_webhooks_for_wallet_by_topic(
+    topic: topics = "connections",
```

Comment:
Thinking of a logical default topic, I can't think of one. Why connections over credentials, etc...? Maybe we should make the topic value required?

Reply:
TBH I agree with you here and am also in favour of making topic required.

I did connections because it's sort of the first thing one establishes in any kind of flow but really not much more reason to this than that. so I'll remove the default